cmd/yggctl/actions.go

```diff
@@ -42,6 +56,81 @@ func generateControlMessageAction(c *cli.Context) error {
 	return nil
 }
 
+func messageJournalAction(ctx *cli.Context) error {
+	// Set the worker message truncate length from the user-provided arguments
+	truncateLength := ctx.Int("truncate-message")
```

Comment:
Using a `cli.UintFlag` here will enable you to avoid checking the flag value for negativity.

Reply:
I thought I made this change... I guess that commit got lost somewhere in the git frenzy. Thanks for pointing this out :+1: